app/filtering.js

```diff
@@ -468,7 +468,33 @@ module.exports.init = () => {
   })
 }
 
-module.exports.isResourceEnabled = (resourceName) => {
+module.exports.isResourceEnabled = (resourceName, url) => {
```

Comment:
i think this needs to have some sort of parameter indicating whether this is being called for a private partition (see `registerPermissionHandler` for example) so that it can check `temporarySiteSettings` instead of `siteSettings` if necessary.

the general principles i was using:
* siteSettings apply to all tabs, including private
* temporarySiteSettings apply to only private tabs
* siteSettings is persistent
* temporarySiteSettings is cleared on browser close